cetmix_tower_server/tests/common.py

```diff
@@ -208,6 +208,30 @@ def setUp(self, *args, **kwargs):
             }
         )
 
+        # Command
+        self.command_run_flight_plan = self.Command.create(
```

Comment:
We are using this in the `test_plan.py' only.  Probably it would be reasonable to move it to the `setUp` of the corresponding class.